src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -691,6 +694,10 @@ def add_fluorescence_traces(
     NWBFile
         The nwbfile passed as an input with the fluorescence traces added.
     """
+    if iterator_options is None:
```

Comment:
What about using the or notation in these cases here that I have seen you using in other places?
`iterator_options = iterator_options or dict()`

Reply:
Sure